### DISCUSSION TITLE
Unable to pull latest alpine docker image

### DISCUSSION THREAD

**Discussion by irvintim** [vectordotdev/vector#16056] [Q&A] [answered]
I'm trying to pull the latest 0.27 Alpine docker image, but I'm get an authorization error:

```
$ docker pull timberio/vector:0.27.X-alpine
Error response from daemon: Head "https://registry-1.docker.io/v2/timberio/vector/manifests/0.27.X-alpine": unauthorized: please use personal access token to login
```

**Comment by spencergilbert** [comment, marked as answer]
Hey @irvintim - are you still seeing this, I don't believe I'm signed in at all and it's working for me today:

```shell
❯ docker pull timberio/vector:0.27.X-alpine
0.27.X-alpine: Pulling from timberio/vector
90cda3b7c325: Pull complete
5f3e186d8cc6: Pull complete
fbf3b84ba7c8: Pull complete
157fb162f595: Pull complete
6e3737fe02b4: Pull complete
4f4fb700ef54: Pull complete
Digest: sha256:e307cc9eebff17e81ac384bd9285511620326278bd3df112b9fa5abacbd02450
Status: Downloaded newer image for timberio/vector:0.27.X-alpine
docker.io/timberio/vector:0.27.X-alpine
```

**Reply by irvintim** [reply]
So wierd, my docker client was logged into dockerhub for our private repo, and that was preventing me from accessing the timberio public repo.  After I did a `docker logout` it started working.  That's new (to me).  Thanks for checking on your side.

**Reply by spencergilbert** [reply]
Definitely strange, I haven't seen that before either. Thanks for reporting a possible issue!
